assets/js/common/ComposedFilter/ComposedFilter.jsx

```diff
@@ -1,11 +1,18 @@
 import React, { useState } from 'react';
 import Button from '@common/Button';
 import Filter from '@common/Filter';
+import DateFilter from '@common/DateFilter';
 
-const renderFilter = (key, { type, ...filterProps }, value, onChange) =>
-  type === 'select' ? (
-    <Filter key={key} {...filterProps} value={value} onChange={onChange} />
-  ) : null;
+const renderFilter = ({ type, ...filterProps }, value, onChange) => {
```

Comment:
**issue:** I guess we lost `key` during a rebase. This is causing test failures.

Reply:
Good catch, thanks! Fixed in https://github.com/trento-project/web/pull/2849/commits/57e72fdff71b304074a0503de8950cc50217c776